src/main/java/com/expedia/www/spring/cloud/sleuth/haystack/reporter/configuration/HaystackReporterAutoConfiguration.java

```diff
@@ -72,6 +76,14 @@ public HaystackReporter spanReporter(String serviceName, List<Client> clients) {
                     settings.getClient().getGrpc().getPort()).build());
         }
 
+        if (settings.getClient().getHttp() != null) {
+
+            final Map<String, String> headers = new HashMap<>();
+            headers.put("client-id", serviceName);
```

Comment:
can you add the support for adding custom headers from the configuration? Something like
```yaml
    haystack:
      enabled: true
      client:
        http:
          endpoint: http://localhost:80/span
          headers: 
                 - key: some-key
                    value: some-value
```